src/stylesheets/components/_components.accordion.scss

```diff
@@ -0,0 +1,39 @@
+.c-accordion {
+}
+
+.c-accordion__header {
+  display: flex;
+  @include inuit-font-size($font-size-p,$line-height-p);
+  width: 120px;
+  color: $tundora;
+  align-items: center;
+}
+
+.c-accordion__icon {
+  margin-left: auto;
+  @include mq ($from: tablet) {
+    display: none;
+  }
+
+  &:hover,
+  &:focus {
+    fill: $celery;
+  }
+}
+
+.c-accordion__list {
+  margin-left: $inuit-global-spacing-unit;
+  @include mq ($from: tablet) {
+    margin-left: 0;
+    display: block !important;
+  }
+}
+
+.c-accordion__panel_text {
+  color: $celery;
+  @include inuit-font-size($font-size-p,$line-height-p);
+}
+
+.isCollapsed {
```

Comment:
Update to `is-collapsed` as per previous comment, also, recommend moving this to `.c-accordion__list` as

```
.c-accordion__list {
....
  &.is-collapsed { .. }
}
```